flake.nix

```diff
@@ -284,6 +284,23 @@
           ];
         };
 
+      homeManagerModules =
+        let
+          inherit (nixpkgs) lib;
+        in
+        # Drop this alias after 26.05
+        lib.mkIf (!lib.oldestSupportedReleaseIsAtLeast 2605) (
+          # Throw after 25.11, warn before
+          (
+            if lib.oldestSupportedReleaseIsAtLeast 2511 then
+              builtins.warn
+            else
+              builtins.throw
+          )
+            "stylix: flake output `homeManagerModules` has been renamed to `homeModules`"
+            self.homeModules
```

Comment:
I know this was my suggestion, but I didn't test if `(throw "foo") "bar"` works as intended and throws before attempting to apply `"bar"`.

It should work, but worth checking.